week2/utilities/data_prepper.py

```diff
@@ -242,13 +245,43 @@ def __log_ltr_query_features(self, query_id, key, query_doc_ids, click_prior_que
         feature_results["sku"] = []
         feature_results["salePrice"] = []
         feature_results["name_match"] = []
-        rng = np.random.default_rng(12345)
-        for doc_id in query_doc_ids:
-            feature_results["doc_id"].append(doc_id)  # capture the doc id so we can join later
+        feature_results["salesRankShortTerm"] = []
```

Comment:
While this works, would be good to not have to hardcode these, so that we don't have to change code when changing features.